lib/widget/dialogs/cannot_open_file_dialog.dart

```diff
@@ -3,20 +3,25 @@ import 'package:localsend_app/gen/strings.g.dart';
 import 'package:localsend_app/util/platform_check.dart';
 import 'package:localsend_app/widget/dialogs/custom_bottom_sheet.dart';
 import 'package:routerino/routerino.dart';
+import '../../provider/receive_history_provider.dart';
 
 class CannotOpenFileDialog extends StatelessWidget {
   final String path;
 
   const CannotOpenFileDialog({required this.path, super.key});
 
-  static void open(BuildContext context, String path) {
+  static void open(BuildContext context, String path, String? fileId, ReceiveHistoryNotifier? filesRef) {
     if (checkPlatformIsDesktop()) {
       showDialog(
         context: context,
         builder: (_) => AlertDialog(
           title: Text(t.dialogs.cannotOpenFile.title),
           content: Text(t.dialogs.cannotOpenFile.content(file: path)),
           actions: [
+            TextButton(
+            onPressed: () => { filesRef!.removeEntry(fileId!), context.pop() },
```

Comment:
In Dart/Flutter, `=>` is used for single statements. You are returning a `Set` containing 2 items here.

You need something like this:

```dart
onPressed: () {
  a();
  b();
}
```

Reply:
The Sets approach worked, but I agree that is not the most straightforward. I'll commit the changes!

Reply:
Yep, during `Set` creation, the necessary method calls are taken. That's why it worked :)